go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -39,6 +40,12 @@ import (
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
+var (
+	throttlerRunning  = stats.NewGauge("TxThrottlerRunning", "transaction throttler running state")
+	requestsTotal     = stats.NewCounter("TxThrottlerRequests", "transaction throttler requests")
+	requestsThrottled = stats.NewCounter("TxThrottlerThrottled", "transaction throttler requests throttled")
```

Comment:
you would need to use use `env.exporter` to create new stats as the metrics will conflict for multi-schema support.
you can refer to `go/vt/vttablet/tabletserver/schema/engine.go`